subsys/bluetooth/host/att.c

```diff
@@ -2998,6 +2998,19 @@ static void att_chan_attach(struct bt_att *att, struct bt_att_chan *chan)
 	sys_slist_prepend(&att->chans, &chan->node);
 }
 
+#if defined(CONFIG_BT_EATT)
+static void cap_eatt_mtu(struct bt_l2cap_le_chan *le_chan)
+{
+	if (le_chan->tx.mtu > le_chan->rx.mtu) {
+		BT_DBG("chan %p (0x%04x): saturating TX MTU to RX MTU (%d)",
```

Comment:
Little unsure of the word `saturating` here, but given the context it works :)

Reply:
yeah was a bit unsure too. `cap`, `limit` or `saturate`. But I think saturate is really what's happening :)